overlays/haskell.nix

```diff
@@ -257,37 +257,38 @@ final: prev: {
                 '';
               };
               # This is very big, and cheap to build: prefer building it locally
```

Comment:
I just realised I forgot to remove this comment (it's also quite slow for the main hackage)